### ISSUE TITLE
Access static images?

### ISSUE THREAD

**Issue by garmeeh** [fwouts/react-screenshot-test#226]
Hey 👋 

Wondering have you thought of the following and have any advice:

My use case is I am using Storybook alongside a Next.js application to build out a component library. As part of this, I am adding images/illustrations for components to a folder that will be publicly accessible. [Next.js Static File Serving](https://nextjs.org/docs/basic-features/static-file-serving)

To allow Storybook to request these files I start up a simple http server in that directory and pass set this URL as the CDN URL for the components. 

```
http-server ./public -p 8081
```

Code example:

```
<img src={`${props.cdnUrl}/my-image.png`} alt="" />
```

CDN URL Values:
```
local dev = ''
storybook = http://192.168.1.22:8081 
jest = http://192.168.1.22:8081
```

I am guessing Docker is unable to access this URL as it is on the host machine 🤔

**Comment by garmeeh** [comment]
Hey @fwouts, that sounds like it could work for this case. 

How would that work then? 

Would I still need to be running a local http server for the `public` directory, or would you somehow mount it in Docker?

**Comment by fwouts** [comment]
You wouldn't have to run a local HTTP server, it would be handled by `ReactComponentServer`. That one doesn't run in Docker, it's only the renderer that does :)

**Comment by fwouts** [comment]
@garmeeh Please check `images.screenshot.tsx` in https://github.com/fwouts/react-screenshot-test/pull/227/files#diff-d97e35aa865c5f7eb093a40e5327c4e5. Would this syntax work for you?

**Comment by garmeeh** [comment]
@fwouts This looks good to me 👌 🔥

**Comment by fwouts** [comment]
Now available in v2.1.0 :)

**Comment by garmeeh** [comment]
Hey @fwouts thanks so much for this. It worked a treat 👌 

Juggling projects at work so only got to test this out today.
